stored_procedures/README.md

```diff
@@ -14,7 +14,10 @@ CALL bqutil.procedure.GetNextIds(10, next_ids);
 
 ## Stored Procedures
 
-* [GetNextIds](#GetNextIds)
+* [GetNextIds](#getnextidsid_count-int64-out-next_ids-array)
+* [chi_square](#chi_squaretable_name-string-independent_var-string-dependent_var-string-out-result-structx-float64-dof-float64-p-float64)
+* [Benjamini-Hochberg multipletests](#bh_multiple_tests-pvalue_table_name-string-pvalue_column_name-string-n_rows-int64-temp_table_name-string-)
+* [Linear Regression](#linear_regression-table_name-string-independent_var-string-dependent_var-string-out-result-structa-float64-b-float64-r-float64-)
```

Comment:
Last nit, please rename these to the actual udf names
```suggestion
* [bh_multiple_tests](#bh_multiple_tests-pvalue_table_name-string-pvalue_column_name-string-n_rows-int64-temp_table_name-string-)
* [linear_regression](#linear_regression-table_name-string-independent_var-string-dependent_var-string-out-result-structa-float64-b-float64-r-float64-)
```

Reply:
Done!